src/main/java/com/org/jenkins/custom/jenkins/distribution/service/MimeMapping.java

```diff
@@ -0,0 +1,15 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import org.springframework.boot.web.server.MimeMappings;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.boot.web.servlet.server.ConfigurableServletWebServerFactory;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MimeMapping implements WebServerFactoryCustomizer<ConfigurableServletWebServerFactory> {

```

Comment:
Just a minor question should this be 
`public class ServletCustomizer implements WebServerFactoryCustomizer<TomcatServletWebServerFactory>` since we are using a TomCat server as our default

Reply:
Pretend I know nothing. Cause I know nothing.

This solution seems to work but I make no promises. As the expert I defer to you.

Reply:
`TomcatServletWebServerFactory` is a subclass of `ConfigurableServletWebServerFactory` and would probably serve for more specific use cases, but this looks good enough. 
CC @martinda @kwhetstone

Reply:
It is a net improvement, I added a issue to track testing later.